src/app/components/create-post-page/create-post-page.component.scss

```diff
@@ -0,0 +1,32 @@
+main {
+  padding: 100px 0px;
+}
+
+.photo-background-height {
+  height: 300px;
+}
+
+.photo-background {
+  width: 300px;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center; 
+}
+
+textarea {
+  height: 300px;
+}
+
+input[type="file"] {
+  display: none;
+}
+
+.hand-cursor {
+  cursor: pointer;
+}
+
+@media screen and (max-width:350px) {
+  .photo-background  {
+    width: 100%;
+  }
+}
```

Comment:
Can't all the height and width fixed values be done with Bootstrap? I believe there are utilities for it as well as for the display: none.

Reply:
Do you know how the with can be set to 100% on small screens? I cant find it

Reply:
uhhh I think just use col-sm-12. Thought you could do it with width in a similar way like w-sm-100 or sm-w-100, but  I see there is no option like that available.